### ISSUE TITLE
Marketplace Integration from On-demand-api repo

### ISSUE THREAD

**Issue by galenatjpl** [unity-sds/unity-on-demand#29]
**Reference Examples from SPS Marketplace definition:**
https://github.com/unity-sds/unity-marketplace
https://github.com/unity-sds/unity-marketplace/blob/main/applications/unity-sps/0.1/metadata.json

**Acceptance Criteria:**
- [x] Marketplace definition
- [x] Update/created terraform scripts
- [ ] Necessary updates to the JSON deployment metadata
- [ ] Verify on-demand API can get properly deployed via the MC. (BLOCKED CURRENTLY)
  (blocked by https://github.com/unity-sds/unity-cs/issues/225)

**Comment by galenatjpl** [comment]
https://unity-sds.gitbook.io/docs/developer-docs/common-services/docs/users-guide/deployment/unity-marketplace

**Comment by galenatjpl** [comment]
I'm going to re-assign this ticket to me, and move it to a "Future Work" (backlog this ticket).
@pymonger @GodwinShen @mike-gangl this is part of the deprecation of the pre-pivot-to-Initiators body of work.
